policyengine_us/variables/gov/states/oh/tax/income/credits/lump_sum_distribution/oh_distribution_credit_eligible_person.py

```diff
@@ -0,0 +1,16 @@
+from policyengine_us.model_api import *
+
+
+class oh_distribution_credit_eligible_person(Variable):
+    value_type = bool
+    entity = Person
+    label = "Eligible person for the Ohio lump sum distribution credit"
+    definition_period = YEAR
+    reference = (
+        "https://tax.ohio.gov/static/forms/ohio_individual/individual/2021/pit-it1040-booklet.pdf#page=29",
+    )
+    defined_for = StateCode.OH
+
+    def formula(person, period, parameters):
+        p = parameters(period).gov.states.oh.tax.income.credits
+        return person("age", period) >= p.senior_citizen.age_threshold
```

Comment:
Can you add a legal code reference showing that it points to the senior citizen credit part of the law? The form isn't clear; they could be defining 65 independently.